src/server_manager/messages/master_messages.json

```diff
@@ -501,6 +509,30 @@
     "message": "Click 'Create'.",
     "description": "This string appears within the server setup view as an item of a list that provides instructions to configure a firewall in Google Cloud Platform. 'Create' should be translated and included untranslated in parentheses."
```

Comment:
`'Create' should *not* be translated`?

Reply:
That's not consistent with the other instructions in the messages.

Reply:
ok